README.md

```diff
@@ -58,3 +58,14 @@ Additional NOTE: the `structs` defined in this project match those in staging. T
 To run all tests use the `cargo test` command. If you want to run a specific subset of tests, then add additionally any info regarding the path of the tests and `cargo` will match it.
 
 Example: `cargo test converters` will run every tests under the `converters` module. Matching occurs also at a test name level, so by writing the full name of a test, that single test can be run.
+
+Additionally, for cache testing, we have included a script that fills up the cache as it would happen in production. You can find the script in `./scripts/load_tester/start.py`. To run the script, use the following commands: 
+
+```shell
+$ alias python=python3
```

Comment:
I would simplify these steps:
- Remove `$` (so that I can copy paste full line)
- Skip python alias ... not really required
- Skip `stat` step, just directly call `python3 -m venv venv`